packages/apputils/src/tokens.ts

```diff
@@ -101,11 +101,36 @@ export interface IThemeManager {
    */
   readonly theme: string | null;
 
+  /**
+   * Get the name of the preferred light theme.
+   */
+  readonly preferredLightTheme: string | null;
+
+  /**
+   * Get the name of the preferred dark theme.
+   */
+  readonly preferredDarkTheme: string | null;
+
+  /**
+   * Get the name of the preferred theme.
+   */
+  readonly preferredTheme: string | null;
+
   /**
    * The names of the registered themes.
    */
   readonly themes: ReadonlyArray<string>;
 
+  /**
+   * Get the names of the registered light themes.
+   */
+  readonly lightThemes: ReadonlyArray<string>;
```

Comment:
Could this new property affect downstreams implementing an `IThemeManager`?

Reply:
For example Voila has its own `IThemeManager`: https://github.com/voila-dashboards/voila/blob/3ac569a4f4df15d094f7cad7c22c9934231023a6/packages/voila/src/plugins/themes/thememanager.ts#L36

Reply:
I second @jtpio all those new attributes should support being `undefined` otherwise this will break overridden token `IThemeManager`.